openedx/core/djangoapps/user_authn/views/utils.py

```diff
@@ -177,3 +181,23 @@ def get_auto_generated_username(data):
     # We generate the username regardless of whether the name is empty or invalid. We do this
     # because the name validations occur later, ensuring that users cannot create an account without a valid name.
     return f"{username_prefix}_{username_suffix}" if username_prefix else username_suffix
+
+
+def remove_disabled_country_from_list(countries: Dict) -> Dict:
+    """
+    Remove disabled countries from the list of countries.
+
+    Args:
+    - countries (dict): List of countries.
+
+    Returns:
+    - dict: Dict of countries with disabled countries removed.
+    """
+    enabled_countries = {}
+    if ENABLE_COUNTRY_DISABLING.is_enabled():
```

Comment:
wouldn't it be more efficient to just for all keys in the `DISABLED` list, do a `del enabled_countries[code]`?